gpu/cmd/webgpuinfo/README.md

```diff
@@ -0,0 +1,8 @@
+# webgpuinfo
+
+This tool prints out information about your available WebGPU devices.  Do this:
+
+```sh
+go run cogentcore.org/core/gpu/cmd/webgpuinfo@latest
```

Comment:
As I said previously, `@latest` is rarely the version you actually want while developing, so either change this to `@main` or get rid of it; you don't need to clarify how to run a Go command.